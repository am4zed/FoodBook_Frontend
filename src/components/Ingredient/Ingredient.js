import React from 'react';

const Ingredient = (props) => {
  return (
  <>
      <li>{props.line}</li>
  </>
)};

export default Ingredient;
