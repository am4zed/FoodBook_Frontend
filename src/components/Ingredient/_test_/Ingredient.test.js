import React from 'react';
import Ingredient from '../Ingredient';

describe("Ingredient", () => {

  void test("snapshot", () => {
    const wrapper = shallow(<Ingredient />)
    expect(wrapper).toMatchSnapshot();
  });
})
