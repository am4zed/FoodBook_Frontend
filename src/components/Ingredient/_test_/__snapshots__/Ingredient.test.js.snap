// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ingredient snapshot 1`] = `
<Fragment>
  <li>
     
     
  </li>
</Fragment>
`;
